content/doc/pipeline/tour/agents.adoc

```diff
@@ -90,4 +90,7 @@ Mixing and matching different containers, or other agents, allows quite a lot
 of flexibility when executing a Pipeline, for more configuration options,
 **link:../environment[continue to "Using environment variables."]**
 
-include::doc/_feedback-footer.adoc[]
+''''
```

Comment:
Did you mean `'''`? 
http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#horizontal-rules-and-page-breaks